### DISCUSSION TITLE
Error while importing 'franc' package

### DISCUSSION THREAD

**Discussion by SkifAlef** [adonisjs/core#3944] [Help] [answered]
Hey everyone,

I'm trying to import [franc](https://github.com/wooorm/franc) package:
```
import {franc} from 'franc-min';
getLang(): string {
    return franc(this._draft.title);
}
```

But instead of getting most probable string language I'm getting this error:
```
Error [ERR_REQUIRE_ESM]: require() of ES Module E:\dev\skiflib3\node_modules\franc-min\index.js from E:\dev\skiflib3\app\Utilities\EpubParser.ts not supported.
Instead change the require of index.js in E:\dev\skiflib3\app\Utilities\EpubParser.ts to a dynamic import() which is available in all CommonJS modules.
```

I've tried to dynamically import franc in function like `const {franc} = await import('franc')` buto to no avail.

So is it adonis related error, franc related error or me related error? :)

**Comment by RomainLanz** [comment, marked as answer]
Hey @SkifAlef! 👋🏻 

You should use a version that is not ESM only.

**Reply by SkifAlef** [reply]
Ah, I see, thanks :)
